lepton/qemu.go

```diff
@@ -228,6 +310,44 @@ func (q *qemu) addOption(flag, value string) {
 	q.flags = append(q.flags, option)
 }
 
+func (q *qemu) addAccel() {
+	// hv_support should not throw errors. When error/log handling is
+	// implemented, then errors in detecting hardware acceleration support
+	// should be reported. Here we treat any error in detecting hardware
+	// acceleration the same as detecting no hardware acceleration - we don't
+	// add the flag.
+
+	const hvfSupportedVersion = "2.12" // https://wiki.qemu.org/ChangeLog/2.12#Host_support
+	const accelFlagVersion = "2.12"
+	qemuVersion, err := qemuVersion()
+	if err != nil {
+		return
+	}
+	ok, err := hv_support()
+	if !(ok && err == nil) {
+		return
+	}
+	if runtime.GOOS == "darwin" {
+		if ok, _ := q.versionCompare(qemuVersion, hvfSupportedVersion); ok {
+			q.addOption("-accel", "hvf")
+			q.addOption("-cpu", "host")
+			return
+		}
+	}
+	if runtime.GOOS == "linux" {
+		ok, err := kvmGroupPermissions()
+		if !(ok && err == nil) {
+			return
+		}
+		if ok, _ := q.versionCompare(qemuVersion, accelFlagVersion); ok {
+			q.addOption("-accel", "kvm")
+		} else {
+			q.addFlag("-enable-kvm")
```

Comment:
The argument accel=kvm of the -machine option is equivalent to the -enable-kvm option. Just use `enable-kvm` and you can get rid of the version check for linux too.
https://wiki.archlinux.org/index.php/QEMU